### PR TITLE
React: add icons

### PR DESCRIPTION
**Pull request by agaspardcilia**
This icons were imported but never actually added to the icon library.

- Please make sure the below checklist is followed for Pull Requests.

- [ ] [Travis tests](https://travis-ci.org/jhipster/generator-jhipster/pull_requests) are green
- [x] Tests are added where necessary
- [x] Documentation is added/updated where necessary
- [x] Coding Rules & Commit Guidelines as per our [CONTRIBUTING.md document](https://github.com/jhipster/generator-jhipster/blob/master/CONTRIBUTING.md) are followed

### CODE REVIEW

File: generators/client/templates/react/src/main/webapp/app/config/icon-loader.ts.ejs [hunk 1]
```diff
@@ -76,6 +76,8 @@ export const loadIcons = () => {
     faFlag,
     faBell,
     faHome,
+    faRoad,
```
Comment (vishal423) [inline generators/client/templates/react/src/main/webapp/app/config/icon-loader.ts.ejs]: isn't this duplicate of last entry?

Comment (agaspardcilia) [reply generators/client/templates/react/src/main/webapp/app/config/icon-loader.ts.ejs]: It is, I meant to add `faSearch`. :sweat_smile:

File: generators/client/templates/react/src/main/webapp/app/config/icon-loader.ts.ejs [hunk 2]
```diff
@@ -76,6 +76,8 @@ export const loadIcons = () => {
     faFlag,
     faBell,
     faHome,
-    faRoad
+    faRoad,
+    faTimesCircle,
+    faSearch
```
Comment (vishal423) [inline generators/client/templates/react/src/main/webapp/app/config/icon-loader.ts.ejs]: shouldn't this be conditional as it's used only when elasticsearch is used

Comment (agaspardcilia) [reply generators/client/templates/react/src/main/webapp/app/config/icon-loader.ts.ejs]: I don't know, I don't think it's really that important. I mean do we really need to start making that file complicated just to save a few kb. wdyt?

Comment (deepu105) [reply generators/client/templates/react/src/main/webapp/app/config/icon-loader.ts.ejs]: I guess one icons doesn't harm that much, we can start adding conditions if there are many

**Comment by deepu105** [comment]
@dcaravana where is it used? would you like to do a PR to add it

**Comment by dcaravana** [comment]
Sorry, it's used as icon of Accounts/Sessions menu item.
I'd love to do a PR but I'm afraid I'll not have the time to do so, let me know if you need that anyway.

**Pull request merged**